### PR TITLE
Get current fee window

### PR DESCRIPTION
**Pull request by epheph**

### CODE REVIEW

**Comment by epheph** [comment]
@stephensprinkle looks like this

```
$ node test-client.js payloads/getFeeWindowCurrent.json

payloads/getFeeWindowCurrent.json
{
  "id": 4,
  "jsonrpc": "2.0",
  "method": "getFeeWindowCurrent",
  "params": {
    "universe": "0x000000000000000000000000000000000000000b"
  }
}

{
  "id": 4,
  "result": {
    "feeWindow": "0x2000000000000000000000000000000000000000",
    "feeWindowID": 457,
    "universe": "0x000000000000000000000000000000000000000b",
    "startBlockNumber": 1500001,
    "startTime": 1509065473,
    "endBlockNumber": null,
    "endTime": 1599999999,
    "fees": 0
  },
  "jsonrpc": "2.0"
}
```

File: test/server/getters/get-fee-window-current.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+"use strict";
```
Comment (pgebheim) [inline test/server/getters/get-fee-window-current.js]: This file doesn't seem to test getFeeWindowCurrent in any way...

Comment (epheph) [reply test/server/getters/get-fee-window-current.js]: uhhhh, yea.

Comment (epheph) [reply test/server/getters/get-fee-window-current.js]: Weird, there was a race between my IDE writing the file out and the git add

Comment (epheph) [reply test/server/getters/get-fee-window-current.js]: fixed now

**Pull request merged**